gr-azure-software-radio/grc/azure_software_radio_keyvault.block.yml

```diff
@@ -36,6 +40,7 @@ documentation: |
     Args:
         KeyVault Name: The keyvault name
         key: A key string which maps to a secret stored in the keyvault.
+        default_cred: A DefaultAzureCredential object, other wise an empty string in which case one is generated locally.
```

Comment:
Args here should probably line up with parameter labels, not ids. Either "Default Credential" or "DefaultAzureCredential" depending on how my other comment in this file gets resolved